File: js/day-1/second-solution.js
const secondSolution = (inputStr) => {
  const arr = inputStr.split('\n').map((s) => +s)
  let c = 0
  for (let i = 1; i < arr.length - 2; i += 1) {
    if (arr[i + 2] > arr[i - 1]) { c += 1 }
  }
  return c
}

module.exports = secondSolution

File: py/day-1/second_solution.py
def secondSolution(inputArr):

  def addition(i):
    return int(i)
  arr = map(addition, inputArr)
  c=0
  for i, n in enumerate(arr):
       if (i>0 and i< (len(arr)-2)):
        #  print(i)
         if(arr[i + 2] > arr[i - 1]):
            c += 1 
  return c
